docs/source/specs/openapi.json

```diff
@@ -1305,6 +1331,24 @@
             "schema": {
               "type": "string"
             }
+          },
+          {
+            "in": "query",
+            "name": "users",
+            "required": false,
+            "description": "A comma separated list of users to remove from the group.",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "in": "query",
+            "name": "service-accounts",
+            "required": false,
+            "description": "A comma separated list of service accounts to remove from the group.",
+            "schema": {
+              "type": "string"
+            }
```

Comment:
Do we want these (the `DELETE` spec updates) to be added now? That would generate clients with this capability, which doesn't actually exist (yet).

Reply:
Yes, removed!